node/service/src/grandpa_support.rs

```diff
@@ -279,20 +278,12 @@ mod tests {
 
 		// add 10 blocks
 		push_blocks(10);
-		assert_eq!(
-			client.info().best_number,
-			10,
-		);
+		assert_eq!(client.info().best_number, 10,);
 
 		// we have not reached the pause block
 		// therefore nothing should be restricted
 		assert_eq!(
-			voting_rule.restrict_vote(
-				&*client,
-				&get_header(0),
-				&get_header(10),
-				&get_header(10),
-			),
+			voting_rule.restrict_vote(&*client, &get_header(0), &get_header(10), &get_header(10),),
```

Comment:
with the exception of 1-item tuples, `,)` is an invalid string in our codebase.

Reply:
OK: 372f4f6a.

Reply:
Though I suspect that it is also valid when used appropriately in macros, i.e. https://github.com/paritytech/polkadot/blob/c68aee352b84321b6a5691d38e20550577d60a45/node/network/availability-distribution/src/tests.rs#L35